team_red/models/model.py

```diff
@@ -1,11 +1,46 @@
-from typing import Optional
+from pathlib import Path
+from string import Formatter
+from typing import Any, Dict, List, Optional
 
-from pydantic import BaseModel
+from pydantic import BaseModel, ValidationError, computed_field
 
 
+class PromptConfig(BaseModel):
+    text: str = ""
+    path: Optional[str] = None
+
+    def model_post_init(self, __context: Any) -> None:  # noqa: ANN401
+        if not self.text and self.path is not None:
+            if Path(self.path).exists():
+                with Path(self.path).open("r") as f:
```

Comment:
In QA, the special characters like ä, ü, ö are not recognized in the prompt. Perhaps missing here: 
`encoding="utf-8"`

Reply:
I implemented a prompt loading test case in `test_config` and your intuition seems to be correct (see [test run](https://github.com/caretech-owl/Llama-2-Open-Source-LLM-CPU-Inference/actions/runs/6882729362/job/18721870788)) which works on Linux but fails on Windows. After adding encoding for all `TextMode="r"` open operations, tests pass on Windows as well